kedro-datasets/kedro_datasets/spark/spark_hive_dataset.py

```diff
@@ -20,11 +20,10 @@ class SparkHiveDataSet(AbstractDataSet[DataFrame, DataFrame]):
     This DataSet has some key assumptions:
 
     - Schemas do not change during the pipeline run (defined PKs must be present for the
-    duration of the pipeline)
+    duration of the pipeline).
     - Tables are not being externally modified during upserts. The upsert method is NOT ATOMIC
-
-    to external changes to the target table while executing.
-    Upsert methodology works by leveraging Spark DataFrame execution plan checkpointing.
+    to external changes to the target table while executing. Upsert methodology works by leveraging
+    Spark DataFrame execution plan checkpointing.
```

Comment:
These need to be aligned under the bullet: https://stackoverflow.com/questions/54677795/python-and-sphinx-bullet-point-list-in-multiline-google-style-docstring

Like:
- My long bullet point bblablalball
   continues here